src/config.js

```diff
@@ -1,10 +1,13 @@
-const DB_NAME = 'igbo_api';
-const TEST_DB_NAME = 'test_igbo_api';
+const DB_NAME = "igbo_api";
```

Comment:
It looks like your text editor changed all the single quotes to double-quotes.

Could you change the double-quotes back to single quotes?